src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -1,35 +1,76 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Fab from '@material-ui/core/Fab';
+import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
```

Comment:
I noticed that this is a different style of arrow from that used for the scroll-to-top button, which uses [`KeyboardArrowUpIcon`](https://www.materialui.co/icon/keyboard-arrow-up):
https://github.com/Seneca-CDOT/telescope/blob/9e50064a8a090fd93975e8f037fe67ef5688fa0c/src/frontend/src/components/ScrollToTop/ScrollToTop.jsx#L8

I think using the complimentary [`KeyboardArrowDownIcon`](https://www.materialui.co/icon/keyboard-arrow-down) for consistency, here, might be nice. Thoughts?

Reply:
I tried picking the arrow that looks like the one in our mock ups but I don't think I can actually nail it with MUI icons so I'm changing it to match our other button. Thanks for the suggestion, I think it looks better now.